Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'pry-rails', '~> 0.3.4'
 gem 'rails_12factor', '~> 0.0.3', group: :production
 gem 'rack-cors', '~> 0.4.0', require: 'rack/cors'
 gem 'simple_token_authentication', '~> 1.6.0'
-
+gem 'paranoia'
```

Comment:
Oh I thought that you wanted to know what the gem does, but if you want why is she adding it, I agree with you that there is needed an explanation for it. Maybe you don't need a gem for it.